### PR TITLE
[INV-56] Item create API

### PR DESCRIPTION
**Pull request by HugoKlepsch**
* Shows example usage of complicated schemas and excluding fields
```
curl -X POST \
  http://localhost:1221/item \
  -H 'Content-Type: application/json' \
  -H 'cache-control: no-cache' \
  -d '{
	"location_id": 1,
	"description": "Item description",
	"name": "item name",
	"listed_price": "123.0000000000000000000",
	"purchase_date": "2019-07-28T19:00:21.478300+00:00",
	"purchase_price": "1.0000000000000000000"
}'
```
```
{
    "msg": "Created item"
}
```

### CODE REVIEW

File: server/models.py [hunk 1]
```diff
@@ -30,11 +30,11 @@ class Item(db.Model):
     id = db.Column(db.Integer, nullable=False, autoincrement=True, primary_key=True)
     create_date = db.Column(db.DateTime, nullable=False, default=datetime.utcnow)
     user_id = db.Column(db.Integer, db.ForeignKey(User.__tablename__ + '.id'), nullable=False)
-    location = db.Column(db.Integer, db.ForeignKey(Location.__tablename__ + '.id'), nullable=True)
+    location_id = db.Column(db.Integer, db.ForeignKey(Location.__tablename__ + '.id'), nullable=True)
```
Comment (HugoKlepsch) [inline server/models.py:33]: Fixing this because it is a id, not a location like "pictures" below.

File: server/models.py [hunk 2]
```diff
@@ -53,29 +53,23 @@ class Picture(db.Model):
 
 
 class PictureSchema(JsonApiSchema):
-    _object_class = Picture
```
Comment (HugoKlepsch) [inline server/models.py:56]: These should have been removed when switching from ObjectSchema to DictSchema

**Review by nskoretz — DISMISSED** [review]
aye

**Review by nskoretz — APPROVED** [review]
aye aye

**Pull request merged**